api/v1alpha1/cluster_types.go

```diff
@@ -88,31 +97,96 @@ type Database struct {
 	Etcd Etcd `json:"etcd"`
 }
 
+// ETOSAPI describes the deployment of the ETOS API.
 type ETOSAPI struct {
 	Image `json:",inline"`
+	// The provider secrets are necessary in order to run ETOS the old way and not using the controller.
```

Comment:
"the old way" is not a very clear way to describe the situation IMO.

```suggestion
	// The provider secrets are necessary in order to deploy and run ETOS without using kubernetes controller(s).
```